### ISSUE TITLE
youtube-dl trigger from ts3sounboard bot

### ISSUE THREAD

**Issue by nirnx** [flyth/ts3soundbot#72]
Hello 
I added diffrent url to download without success 
it looks like ts3soundboard triggering youtube-dl like this 
```/usr/local/bin/youtube-dl -f bestaudio -o a --max-downloads 1 --max-filesize 200M --no-playlist --write-info-json --write-thumbnail --no-call-home https://www.deezer.com/playlist/xxxxxxxx```

And error:
```ERROR: requested format not available```

I wonder if it is possible to change "-f bestaudio" format maybe some config.ini ? 

Best Regards ;)

**Comment by flyth** [comment]
Are you sure it's the `-f bestaudio` that's not working and not the `--no-playlist`? However, I'll try to make some of the settings more configurable.

**Comment by nirnx** [comment]
Hello,
Thanks for reply :)

I did some more research and 
``` -f bestaudio   --max-downloads 1 ```
are problematic
Anyway i think the simplest way is to add option to modify YoutubeDLPath with command + args

**Comment by flyth** [comment]
Yeah, max-downloads is however critical right now, due to the way the bot handles the downloads...
